### ISSUE TITLE
Pretrained weights don't work for reconstruct

### ISSUE THREAD

**Issue by Oktai15** [wengong-jin/icml18-jtnn#8]
@wengong-jin Is it okay that pretrained weights (located here: https://github.com/wengong-jin/icml18-jtnn/tree/master/molvae/MPNVAE-h450-L56-d3-beta0.005) reconstruct molecules with accuracy = 0?

**Comment by Oktai15** [comment]
Oh, solved

```nn.Softmax(dim=-1)``` should be if one uses pytorch 0.4 :)
